API.md

```diff
@@ -1,64 +1,283 @@
+  - [Expectations](#expectations)
+    - [expectStatus(code)](#expectstatuscode)
+    - [expectHeader(key, content, options)](#expectheaderkey-content-options)
+    - [expectNoHeader(key)](#expectnoheaderkey)
+    - [expectHeaderContains(key, content, options)](#expectheadercontainskey-content-options)
+    - [expectJSON([path], json)](#expectjsonpath-json)
+    - [expectContainsJSON([path], json)](#expectcontainsjsonpath-json)
+    - [expectJSONTypes([path], schema)](#expectjsontypespath-schema)
+    - [expectBodyContains(content)](#expectbodycontainscontent)
+    - [expectJSONLength([path], length)](#expectjsonlengthpath-length)
+    - [expectMaxResponseTime(ms)](#expectmaxresponsetimems)
+    - [not()](#not)
+    - [Using Paths](#using-paths)
+      - [Testing Nested Objects](#testing-nested-objects)
+      - [Testing All Objects in an Array](#testing-all-objects-in-an-array)
+      - [Testing One Object in an Array](#testing-one-object-in-an-array)
+  - [useApp()](#useapp)
+    - [Example Use:](#example-use)
+      - [Express Application:](#express-application)
+      - [IcedFrisby Test:](#icedfrisby-test)
+  - [Global Setup](#global-setup)
+    - [request.baseUri](#requestbaseuri)
+    - [request.headers](#requestheaders)
+    - [request.json](#requestjson)
+    - [request.inspectOnFailure](#requestinspectonfailure)
+    - [failOnMultiSetup](#failonmultisetup)
+    - [useApp](#useapp-in-globalsetup)
+    - [Resetting `globalSetup`](#resetting-globalsetup)
+  - [Helpers](#helpers)
+    - [before()](#before)
+    - [after()](#after)
+    - [finally()](#finally)
+    - [afterJSON()](#afterjson)
+    - [only()](#only)
+    - [timeout(ms)](#timeoutms)
+    - [retry(count, backoff)](#retrycount-backoff)
+    - [baseUri(uri)](#baseuriuri)
+    - [waits(ms)](#waitsms)
+    - [exceptionHandler(function)](#exceptionhandlerfunction)
+  - [Inspectors](#inspectors)
+    - [inspect(cb)](#inspectcb)
+    - [inspectRequest(message)](#inspectrequestmessage)
+    - [inspectResponse(message)](#inspectresponsemessage)
+    - [inspectHeaders(message)](#inspectheadersmessage)
+    - [inspectJSON(message)](#inspectjsonmessage)
+    - [inspectBody(message)](#inspectbodymessage)
+    - [inspectStatus(message)](#inspectstatusmessage)
+    - [Send Raw JSON or POST Body](#send-raw-json-or-post-body)
+
+## The Basics
+
+Every frisby request begins with `create(..)` and ends with a `toss()`.
+
+### create(msg)
+
+| Parameter | Description | Required |
+| --------- | ----------- | -------- |
+| msg | A string used to name the test when it's wrapped in mocha for execution at runtime | Yes |
+
+Used to create an instance of IcedFrisby that is used to send 1 request and receive the response.
+
+```javascript
+frisby.create('a test')
+    .get('http://example.com')
+    .expectStatus(200)
+    // any number of additional expect statements here
+    .toss();
+```
+
+### toss()
+
+Used to complete the list of commands and issue the request.
```

Comment:
> Complete the list of commands and register the test with Mocha.

Reply:
Much better. I knew that wasn't quite accurate when I wrote it, but couldn't word it better.